chunky/src/java/se/llbit/chunky/plugin/ContextMenuTransformer.java

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright (c) 2022 Chunky contributors
+ *
+ * This file is part of Chunky.
+ *
+ * Chunky is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * Chunky is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ * You should have received a copy of the GNU General Public License
+ * along with Chunky.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+package se.llbit.chunky.plugin;
+
+import javafx.scene.control.ContextMenu;
+
+import java.util.function.Consumer;
+
+@FunctionalInterface
+public interface ContextMenuTransformer extends Consumer<ContextMenu> {
```

Comment:
These could be even more powerful if they were _actual_ transformers, ie. `Function<ContextMenu, ContextMenu>` because then they could even replace the menu entirely. :thinking: Not sure how much power we want to give to plugins though.